### DISCUSSION TITLE
strongswan with aws lightsail server

### DISCUSSION THREAD

**Discussion by john047** [strongswan/strongswan#361] [Q&A] [answered]
Hello,

I configured strongswan on Ubuntu 20.04.2 LTS. Connection is establish between server and client but a client doesn't have access to Internet.
Aws lightsail server has two IP addresses - private and public. The private ip assigned to eth0 interface. The public ip i can't see in system only through brower (aws console).
Therefore I configured ufw with my private ip (eth0).
In ipsec.conf I specified my public ip.
How to fix the lack of Internet on the client?
Help, please!

Thanks in advance!

**Comment by Thermi** [comment, marked as answer]
Look and apply the advice from the wiki here: https://wiki.strongswan.org/projects/strongswan/wiki/ForwardingAndSplitTunneling
